easybuild/easyconfigs/m/MDTraj/MDTraj-1.9.1-intel-2017b-Python-3.6.3.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'PythonPackage'
+
+name = 'MDTraj'
+version = '1.9.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://mdtraj.org/1.9.0/'
+description = "Read, write and analyze MD trajectories with only a few lines of Python code."
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+source_urls = ['https://github.com/mdtraj/mdtraj/archive']
+sources = ['%(version)s.tar.gz']
+checksums = ['3e27cd3406521a1c9f781efc0bd41efebb9f2334a9bb09cb1d9ca8edca419d09']
+
+dependencies = [
+    ('Python', '3.6.3'),
+    ('zlib', '1.2.11'),
+]
+
+# The unit tests of MDTraj are a pain to get to work: they require
+# a massive number of extra dependencies. See
+# https://github.com/mdtraj/mdtraj/blob/master/devtools/conda-recipe/meta.yaml
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/%(namelower)s-%(version)s-py%(pyshortver)s-linux-x86_64.egg/%(namelower)s'],
```

Comment:
This line is too long, so it should be trimmed somehow (that's also why the tests are failing in Travis, we're stricter w.r.t. style now in easyconfigs).

You can just check for `lib/python%(pyshortver)s/site-packages` here, since EasyBuild already does an `import mdtraj` check.

Reply:
Good. How does easybuild now that it should try `import mdtraj`? The module name is not necessarily related to the package name.

Reply:
By default is uses `name.lower()`, you can override it with:

```python
options = {'modulename': 'foo'}
```

Reply:
Allright! Thanks. In that case, I can remove this line from the yaff eb file.